Userland/Libraries/LibWeb/Painting/Paintable.h

```diff
@@ -266,6 +268,7 @@ class Paintable
     bool m_absolutely_positioned : 1 { false };
     bool m_floating : 1 { false };
     bool m_inline : 1 { false };
+    bool m_selected : 1 { false };
```

Comment:
it's not possible to store selection state inside paintable tree because then it will be reset after each relayout.

Reply:
aha, so `update_selection()` is invoked in the end of `Document::update_layout()`. it's probably ok then, but, still, would much better if we could hold selection state in a DOM tree.

Reply:
Selection can be performed on fragments located in the Layout/Paintable tree, meaning the selection state cannot be stored in the DOM tree. As mentioned in [this commit](https://github.com/LadybirdBrowser/ladybird/commit/1987318cc25ff01de339e9f5a419212b78d37212), selection is paint-related information, so I've kept m_selected in `Paintable.h`.